spec/departure_comparator_spec.rb

```diff
@@ -44,12 +44,21 @@
         expected_time =
           "Expected SDT: #{email_format @gtfs_time}"
 
-        report_missing_route @route_number, @headsign, @stop_name, @gtfs_time
+        report_missing_route @route_number, @headsign, @stop_name, @gtfs_time,
```

Comment:
It could happen anytime a route direction is just missing. If anything your statement is the other way around - we won't have alternative headsigns on UMTS routes, because our headsigns match the `InternetServiceDesc` in Avail. But either property could have missing route directions.